.github/workflows/Documentation.yml

```
@@ -10,17 +9,13 @@ on:
   pull_request:
 
 jobs:
-  build:
+  Documenter:
+    name: Documentation
     runs-on: ubuntu-latest
```

The [latest usage](https://github.com/julia-actions/julia-docdeploy) suggests permission to write.
```suggestion
    permissions:
      contents: write
    name: Documentation    
    runs-on: ubuntu-latest
```